extensions/interactions/FractionInput/directives/fraction-input-interaction.component.html

```diff
@@ -10,7 +10,7 @@
          placeholder="{{customPlaceholder !== '' ? customPlaceholder : getPlaceholderText() | translate}}">
 
   <div class="oppia-form-error oppia-fraction-input-error">
-    {{errorMessage}}
+    {{errorMessage | translate}}
```

Comment:
errorMessage should be changed to a variable name that conveys "this is an I18n key"

Reply:
Done